Orange/base.py

```diff
@@ -27,11 +27,15 @@ def __init__(self, preprocessors=None):
         self.preprocessors = list(preprocessors)
 
     def fit(self, X, Y, W=None):
-        raise NotImplementedError(
-            "Descendants of Learner must overload method fit")
+        raise RuntimeError(
+            "Descendants of Learner must overload method fit or "
+            "fit_storage")
```

Comment:
I don't think this change is reasonable or required. Abstract methods `raise NotImplementedError` by convention. Even the soon-to-be-obsolete message is not required. The developer experiencing this exception should know or know where to look what to do.